st2common/st2common/models/db/stormbase.py

```diff
@@ -78,6 +78,23 @@ class StormBaseDB(StormFoundationDB):
     }
 
 
+class StormContentDB(StormBaseDB):
+    """Abstraction for a user content model from a pack."""
+    name = me.StringField(required=True)
+    pack = pack = me.StringField(
+        required=False,
+        help_text='Name of the content pack.',
+        unique_with='name')
+    file_uri = me.StringField(
```

Comment:
On a separate note `entry_point` and `atrifact_uri` are both in SensorTypeDB. AFAICT `entry_point` here is the same as `entry_point` in ActionDB - so why do we need 2 properties?